xml/obs_admin_components.xml

```diff
@@ -7,8 +7,147 @@
+   </imageobject>
+  </mediaobject>
+ </figure>
+ <section>
+   <title>Frontend</title>
+   <para>
+     The OBS Frontend is a Ruby on Rails application that manages the access and
+     manipulation of the OBS data. It provides a web user interface and an
+     application programming interface to do so. Both can be used to create,
+     read update and delete users, projects/packages, requests and other
+     objects. It also implements additional sub-systems like authentication,
+     search or email notifications.
+   </para>
+ </section>
+ <section>
+   <title>Backend</title>
+   <para>
+     The OBS Backend is a collection of Perl applications that manage the source
+     files and build jobs of the OBS.
+   </para>
+   <section>
+     <title>Source Server</title>
+     <para>
+       Maintains the source repository and project/package configurations.
+       It is the only interface for the Frontend and it may forward requests to
+       one of the repository servers. Each OBS installation only has one
+       Source Server.
+     </para>
+   </section>
+   <section>
+     <title>Repository Server</title>
+     <para>
+       A repository server provides access to the repositories of
+       some projects/packages/architectures. It also maintains the
+       build client worker pool, sending jobs to the clients and informing
+       the schedulers about finished jobs. Each OBS installation has
+       one or more repository servers.
+     </para>
+   </section>
+   <section>
+     <title>Scheduler</title>
+     <para>
+       A scheduler maintains all projects/packages of the corresponding
+       repository server, but for just one architecture. It is responsible
+       for starting jobs in the right order, collecting the built
+       binary packages and creating the external repository if a
+       project's repository is completely built. Each OBS installation has
+       one scheduler per available architecture.
+     </para>
+   </section>
+   <section>
+     <title>Dispatcher</title>
+     <para>
+       To be written...
+     </para>
+   </section>
+   <section>
+     <title>Publisher</title>
+     <para>
+       To be written...
+     </para>
+   </section>
+   <section>
+     <title>Worker</title>
+     <para>
+       The workers contact the repository servers for jobs, build
+       the jobs and send the result back to the repository server. Each OBS
+       installation has one or more workers.
+     </para>
+   </section>
+   <section>
+     <title>Signer</title>
+     <para>
+       Sign Packages. To be written...
+     </para>
+   </section>
+   <section>
+     <title>Warden</title>
+     <para>
+       Monitor workers. To be written...
+     </para>
+   </section>
+   <section>
+     <title>Download on Demand</title>
+     <para>
+       To be written...
+     </para>
+   </section>
+   <section>
+     <title>Delta Storage</title>
+     <para>
+       To be written...
+     </para>
+   </section>
+ </section>
```

Comment:
@mlschroe Could you review the backend related docu?^ And ideally add the missing pieces, since you love to write docu:-)